html/modules/custom/docstore/src/MetadataTrait.php

```diff
@@ -273,6 +274,7 @@ public function createTermFromParameters(array $params, Vocabulary $vocabulary,
     $item = [
       'name' => $params['label'],
       'vid' => $vocabulary->id(),
+      'author' => [],
```

Comment:
author is required, so will need some more validation

Reply:
I was getting errors that $item['author'] was undefined a few lines on in `$item['author'][] = [`